app/events/events.go

```diff
@@ -22,7 +22,7 @@ type TbAPI interface {
 	GetUpdatesChan(config tbapi.UpdateConfig) tbapi.UpdatesChannel
 	Send(c tbapi.Chattable) (tbapi.Message, error)
 	Request(c tbapi.Chattable) (*tbapi.APIResponse, error)
-	GetChat(config tbapi.ChatInfoConfig) (tbapi.Chat, error)
+	GetChat(config tbapi.ChatInfoConfig) (tbapi.ChatFullInfo, error)
```

Comment:
Breaking interface change. I guess it would be good to release the current master as the new version so that the new master would get at least different minor number.

Reply:
Well, this interface is not really exposed. I mean, it is public and all of this because I hate "internal" thing, but the actual API, the documented one, is in lib.